cubiscan/cubiscan.py

```diff
@@ -14,11 +15,23 @@ class CubiScan(object):
     port = None
     timeout = None
     registry = None
+    buffer_receive_size = None
+    ssl = None
 
-    def __init__(self, ip_address, port, timeout=30):
+    def __init__(self, ip_address, port, timeout=30, ssl=None,
+                 buffer_recv_size=1024):
         self.ip_address = ip_address
         self.port = port
         self.timeout = timeout
+        self.buffer_receive_size = buffer_recv_size
+
+        if not ssl:
+            self.ssl = False
```

Comment:
I think `None` is a better value to keep in `self.ssl` when it's not set (I know we are used to have `False` everywhere with odoo ;)

Reply:
@guewen done